client/android/app/build.gradle

```diff
@@ -5,17 +5,19 @@ if (localPropertiesFile.exists()) {
         localProperties.load(reader)
     }
 } else {
-  throw new FileNotFoundException("local.properties not found, ensure app is built via Flutter")
+    throw new FileNotFoundException('local.properties not found, ensure app is built via Flutter')
 }
 
 def flutterRoot = localProperties.getProperty('flutter.sdk')
 if (flutterRoot == null) {
-    throw new GradleException("Flutter SDK not found. Define location with flutter.sdk in the local.properties file.")
+    throw new GradleException('Flutter SDK not found. Define location with flutter.sdk in the local.properties file.')
 }
 
 def flutterVersionCode = localProperties.getProperty('flutter.versionCode')
 def flutterVersionName = localProperties.getProperty('flutter.versionName')
 
+def server = "server"
```

Comment:
Move to immediately before `flavorDimensions`, it adds cognitive load to require someone to track down the earlier definition, no reason not to have them together.